utils/kata-manager.sh

```diff
@@ -518,7 +520,9 @@ install_kata()
 	local version_desc="latest version"
 	[ -n "$requested_version" ] && version_desc="version $requested_version"
 
-	info "Downloading $project release ($version_desc)"
+	local latest_version=$(curl -s "$kata_releases_url/latest" | grep "tag_name" | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
```suggestion
	local latest_version=$(curl -sL "$kata_releases_url/latest" | grep "tag_name" | sed -E 's/.*"([^"]+)".*/\1/')
```

It might be worth creating a small function to avoid the duplication, and avoid potential future breakage if we forget to update one of the instances.